### PR TITLE
Parker dev

### PR DESCRIPTION
**Pull request by dpark6060**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,100 @@
...
+#### *Output with **apply_to_X** options, and **topup_only** is false*
+* **topup-corrected-<apply_to_1>.nii.gz**: Topup corrected image from **apply_to_1**
+* **topup-corrected-<apply_to_2>.nii.gz**: Topup corrected image from **apply_to_2**
+*note that if **image-1** and **image-2** are 4D, this gear will also create TOPUP corrected versions of **image-1** and **image-2**.  The output name will be topup-corrected-<image-1>.
+
+
+#### *Output with **QA***
+* **<image_1>-QA-report.png**: QA image from **image_1**
+* **<image_2>-QA-report.png**: QA image from **image_2**
+*Presently only runs on files that have been corrected with topup.  Won't work if **topup_only** is true*
```
Comment (NPann) [inline README.md:69]: Formatting does not render new line here. double space missing after **image_2** I believe

File: README.md [hunk 2]
```diff
@@ -1 +1,100 @@
...
+
+![QA example](https://github.com/flywheel-apps/fsl-topup/blob/parker-dev/QA/nodif_QA_report.png "QA example")
+
+The top image is the original input, with a red overlay of the topup-corrected brain boarder outline.
+
+The bottom image is the topup corrected image, with a red overlay of the original image brain boarder outline.
+
+Ideally, you should see improvement in the shape of the brain's outline, correcting for major distortions such as the one in the front of the brain in the far right
+
+© Copyright 2019, Josh Revision
```
Comment (NPann) [inline README.md:100]: Pun intended? (re the Josh Revision)

File: run.py [hunk 3]
```diff
@@ -11,6 +11,16 @@
 import mri_qa
 import shutil
 
+#### Setup logging as per SSE best practices
+try:
+    log = logging.getLogger('root')
```
Comment (NPann) [inline run.py]: The logger get redefined in each sub function later. I would remove all the other ones.

**Review by NPann — CHANGES_REQUESTED** [review]
Quite a lot to review here
Maybe worth having a run example on ss.ce before merging.

File: README.md (hunk 1 above)
Comment (dpark6060) [reply README.md:69]: https://ss.ce.flywheel.io/#/jobslog/job/5dfbb690f999360023e1bf21

Comment (dpark6060) [reply README.md:69]: ugh yeah pycharm keeps removing trailing spaces :(

**Review by NPann — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (NPann) [reply README.md:69]: weird

File: README.md (hunk 2 above)
Comment (joshicola) [reply README.md:100]: Bwahahhhaaa!

File: common.py [hunk 4]
```diff
@@ -50,7 +49,7 @@ def build_command_list(command, ParamList, include_keys=True):
     return command
 
 
-def exec_command(context, command, shell=False, stdout_msg=None, cont_output=False):
+def exec_command(command, shell=False, stdout_msg=None, cont_output=False):
```
Comment (joshicola) [inline common.py:52]: This form is now in the gear-toolkit.
I don't know if you need the 'environ' for this.

File: common.py [hunk 5]
```diff
@@ -8,8 +8,7 @@
 import subprocess as sp
 import re
 import logging
-
-log = logging.getLogger(__name__)
+log = logging.getLogger()
```
Comment (joshicola) [inline common.py:11]: I suppose we will talk more about how we do logging.

I am in favor of identifying the module and function we are logging from....

I guess more to come.

**Review by joshicola — APPROVED** [review]

**Pull request merged**